content-review/api/src/main/java/org/sakaiproject/contentreview/service/ContentReviewQueueService.java

```diff
@@ -154,6 +154,13 @@ int getReviewScore(Integer providerId, String contentId)
 	 * @return {@code List<ContentReviewItem>}
 	 */
 	List<ContentReviewItem> getAllContentReviewItemsGroupedBySiteAndTask(Integer providerId);
+	
+	/**
+	 * Gets all items grouped by site, order by most recent items descending
+	 * @param providerId the id of content review implementation
+	 * @return {@code List<Object[]>}
+	 */
+	List<Object[]> getContentReviewItemsGroupedBySite(Integer providerId);
```

Comment:
List<Object[]> is a very vague type I would try to improve this, see comment below to improve to List<String[]>